api/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ramendr/ramen/api
 
-go 1.21
+go 1.22.0
```

Comment:
1.22 works:

```
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/ramendr/ramen
 
-go 1.21
+go 1.22
```

Running `go mod tidy` and `make` does not change to 1.22.0. Did yo add it manually or some tool changed it?

Reply:
Changing to 1.22 and running go mod tidy gives me the following error:
go: downloading go1.22 (linux/amd64)
go: download go1.22 for linux/amd64: toolchain not available

Using 1.22.0 is fixing it.

Reply:
Maybe because you don't have go 1.22 installed on you machine?

You can install multiple go versions:
https://go.dev/doc/manage-install

If we depend automatic download of go version for building, we can require the latest version: 1.22.6.

Reply:
I think we need to do this change that Nir suggests.